src/ChooseAGameName/GamePanel.java

```diff
@@ -2,39 +2,33 @@
 
 import java.awt.Color;
 import java.awt.Dimension;
-import java.awt.Graphics;
 import java.awt.GridLayout;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
 import javax.swing.JButton;
 import javax.swing.JLabel;
-import javax.swing.JOptionPane;
 import javax.swing.JPanel;
 import javax.swing.JScrollPane;
 import javax.swing.JTextArea;
-import javax.swing.Timer;
 
 public class GamePanel extends JPanel implements ActionListener {
 
-	// graphics and layout for each page ( game page, and map); maybe handle mouse
-	// clicks(for map)
-
 	// JTextArea storyArea = new JTextArea("hey", 1, 1);
 	JTextArea storyArea = new JTextArea(
 			">Hello, how are you doing today?!\n\n\n\n\n\n\n\nsome text\n\n\n\n\n\n\n\n even more text", 20, 30);
-	JLabel deathCount = new JLabel("Deaths: put num later");
+	StoryPiece storyPiece = new StoryPiece(false, "hey");
+	JLabel deathCount = new JLabel("Deaths: "+storyPiece.deathCount());
 	JScrollPane scroll = new JScrollPane(storyArea);
 	JButton mapButton = new JButton();
 	JLabel map = new JLabel("You're in the map");
 
 	int storyState = 1;
 	int mapState = 2;
 	int currentState = storyState;
-	StoryPiece storyPiece = new StoryPiece(false, "Hey");
-	StoryPiece storyPiece2 = new StoryPiece(true, "Heyy");
-	MapManager mapManager = new MapManager();
+	//StoryPiece storyPiece = new StoryPiece(false, "Hey");
```

Comment:
You can delete this commented-out line if it's no longer necessary.

Reply:
got rid of  it